addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -143,21 +143,15 @@ table.editorbar-panel {
 }
 
 // s_popup in edit mode
-.editor_has_snippets {
-    .s_popup_main {
-        z-index: 1000 !important;
-        left: $o-we-sidebar-width !important;
-        top: var(--o-we-toolbar-height) !important;
-
-        &.s_popup_fixed {
-            .s_popup_full .s_popup_content {
-               left: $o-we-sidebar-width !important;
-            }
+body.editor_enable.editor_has_snippets {
+    .s_popup .modal {
+        top: var(--o-we-toolbar-height);
+        background-color: transparent;
+        &.fade .modal-dialog {
+            transform: none;
         }
-        &.s_popup_fixed_top {
-            .s_popup_content {
-                top: var(--o-we-toolbar-height) !important;
-            }
```

Comment:
Why did this become useless? Did you review the whole layout of the different popups to use standard bootstrap layout? :)

Reply:
Only the layout of the popups related to the s_popup snippet. Why? 
It's useless because these classes don't exist anymore, they have been replaced by BS standard classes.
Should I keep these rules somewhere for compatibility?

Reply:
Not at all, it's cool :) Does this mean that now the modal are always opened with standard bootstrap class and not d-none anymore? It was a problem for LTU's task

Reply:
Yes, indeed! 🥳